incubator/nodejs-express/image/project/Dockerfile

```diff
@@ -28,13 +29,19 @@ RUN apt-get update \
  && echo 'Finished installing dependencies'
 
 # Copy the project
-COPY --chown=node:node . /project
```

Comment:
Does buildah do better  when copying between images than it does when copying from host?

It looks like all the lines from here to `ENV NODE_PATH` could be replaced by a single line:

> COPY --chown=node:node --from=0 /project /project

Reply:
In the first docker image we don't have all the files from the `/project` dir, since we only copy the `package.json` for the project and the user-app. I agree that we could simplify the process by first copying the tars, and `package.json'`s with that one command and then copy the files from the host.

Reply:
The issue we have today is that buildah 1.9.0 does not honour `.dockerignore`, and has a buggy algorithm for iterating over files. We can not move to a higher version (which has this fix) yet because of other issues.

I can move the copy commands in the first stage, however, we will still need those separate copy statements in the first stage then. Would we just move the problem around to the first stage?